### ISSUE TITLE
Feature: Globally zoom object

### ISSUE THREAD

**Issue by elmodor** [drwhut/tabletop-club#114]
**Is your feature request related to a problem? Please describe.**
Sometimes cards or other objects can be very tiny when they are on the table (especially if they have text on them). Currently it's only possible to zoom into cards which are on the players hand.

**Describe the solution you'd like**
Any object should be displayed bigger when hovering over with the mouse and the specific button is pressed, not only cards on the hand.  
Furthermore it should be possible to change the size of the zoomed in popup by using the scrollwheel (to increase/decrease the size of the zoomed in objects).

**Describe alternatives you've considered**
The player would need to zoom into the whole board to read cards on the table if the text on the cards are too tiny to read from the currently perspective.

**Comment by drwhut** [comment]
Cool, I've got a few questions as to how this would work:

- Would the popup show the object in a separate 3D world which you can rotate, zoom in, etc. or would it be like having a 2nd camera which is much closer to the object than the player camera?
- If it is in a separate 3D world, then how would we prevent cheating (i.e. if another player is dragging a card to their hand, and we zoom in before it reaches their hand)?
- What does this functionality solve, that zooming into the table doesn't?

**Comment by elmodor** [comment]
Right, maybe lets stick to cards, tokens and dice. Those have text on them while other objects usually do not.

- Showing only the top side of the card, basically what already happens when the card is in your hand. Then you can't peak under the card.
- Only when the object is stationary. So no shenanigans can happen while a dice is rolling, a card is flipped/drawn.

> What does this functionality solve, that zooming into the table doesn't?

It's basically just some nice to have usability? Of course you can zoom in to the table and pan to the object just to read the text on it. Zoom out, pan to your previous position. 
With this functionality, you can just stay where you are, press the key to quickly get a bigger picture of the card and continue where you left of.

I guess this just depends on the playstyle.

**Comment by drwhut** [comment]
Those are some good ideas to prevent cheating!

I guess yeah, it's just to add another option for looking at something on the table. But since it could be potentially quite tricky to implement for 3D objects like dice (which can't really use the same system as previewing cards, tokens we might be able to get away with since they have a standard UV mapping), I might implement this later down the line, since it only provides an alternative way of doing something that's already possible.

**Comment by elmodor** [comment]
Well you could start with cards (and maybe tokens) and add other further down the line. 
I guess text is the most difficult to read sometimes which is mostly only on cards anyway.
